protostar/commands/invoke_command.py

```diff
@@ -177,7 +178,7 @@ async def invoke(
             inputs=inputs,
             max_fee=max_fee if max_fee is not None else "auto",
             signer=signer,
-            account_address=account_address,
+            account_address=AccountAddress(int(account_address)),
```

Comment:
Unnecessary casting to int

Reply:
https://github.com/software-mansion/protostar/pull/1170#discussion_r1025372518